BitwardenShared/UI/Vault/VaultItem/ViewItem/ViewItemView.swift

```diff
@@ -46,7 +46,26 @@ struct ViewItemView: View {
             send: ViewItemAction.toastShown
         ))
         .toolbar {
+            ToolbarItem(placement: .navigationBarLeading) {
+                closeToolbarButton {
+                    store.send(.dismissPressed)
+                }
+            }
+
             ToolbarItemGroup(placement: .navigationBarTrailing) {
+                if let state = store.state.loadingState.data {
+                    if state.isSoftDeleted {
+                        toolbarButton(Localizations.restore) {
+                            await store.perform(.restorePressed)
```

Comment:
🤔 should we add ViewInspector tests for these?

Reply:
I spent a fair bit of time fiddling with the data to get into the "deleted item" branch, and it's just creating an entire second set of data to get a CipherView that's set as deleted. Should be very doable, just would vastly increase the size of the file, which is I assume the reason it wasn't done before.

Reply:
ah yeah, that makes sense to me!